docs/src/rules/no-implicit-globals.md

```diff
@@ -198,59 +197,59 @@ Examples of **correct** code for this rule with `"lexicalBindings"` option set t
     class Bar {}
 }
 
-(function() {
+(function () {
     const foo = 1;
     let baz;
     class Bar {}
-}());
+})();
 ```
 
 :::
 
 If you intend to create a global `const` or `let` variable or a global `class` declaration, to be used from other scripts,
 be aware that there are certain differences when compared to the traditional methods, which are `var` declarations and assigning to a property of the global `window` object:
 
-* Lexically declared variables cannot be conditionally created. A script cannot check for the existence of
-a variable and then create a new one. `var` variables are also always created, but redeclarations do not
-cause runtime exceptions.
-* Lexically declared variables do not create properties on the global object, which is what a consuming script might expect.
-* Lexically declared variables are shadowing properties of the global object, which might produce errors if a
-consuming script is using both the variable and the property.
-* Lexically declared variables can produce a permanent Temporal Dead Zone (TDZ) if the initialization throws an exception.
-Even the `typeof` check is not safe from TDZ reference exceptions.
+-   Lexically declared variables cannot be conditionally created. A script cannot check for the existence of
+    a variable and then create a new one. `var` variables are also always created, but redeclarations do not
+    cause runtime exceptions.
+-   Lexically declared variables do not create properties on the global object, which is what a consuming script might expect.
+-   Lexically declared variables are shadowing properties of the global object, which might produce errors if a
+    consuming script is using both the variable and the property.
+-   Lexically declared variables can produce a permanent Temporal Dead Zone (TDZ) if the initialization throws an exception.
+    Even the `typeof` check is not safe from TDZ reference exceptions.
 
 Examples of **incorrect** code for this rule with `"lexicalBindings"` option set to `true`:
 
-::: incorrect
+::: incorrect { "sourceType": "script" }
 
 ```js
 /*eslint no-implicit-globals: ["error", {"lexicalBindings": true}]*/
 
-const MyGlobalFunction = (function() {
+const MyGlobalFunction = (function () {
     const a = 1;
     let b = 2;
-    return function() {
+    return function () {
         return a + b;
-    }
-}());
+    };
+})();
 ```
 
 :::
 
 Examples of **correct** code for this rule with `"lexicalBindings"` option set to `true`:
 
-::: correct
+::: correct { "sourceType": "script" }
 
 ```js
 /*eslint no-implicit-globals: ["error", {"lexicalBindings": true}]*/
 
-window.MyGlobalFunction = (function() {
+window.MyGlobalFunction = (function () {
     const a = 1;
     let b = 2;
-    return function() {
+    return function () {
         return a + b;
-    }
-}());
+    };
+})();
```

Comment:
can  you once verify if these prettier changes are needed? lot has changed in this file (spacing, * to -)

Reply:
Blurgh, sorry, this keeps slipping in. As of the latest in https://github.com/eslint/eslint.org/pull/451 the `"sourceType": "script"`s are no longer necessary anyway.